h2o-core/src/main/java/water/Scope.java

```diff
@@ -33,17 +34,12 @@
    *  enter call except for the listed Keys.
    *  @return Returns the list of kept keys. */
   static public Key[] exit(Key... keep) {
```

Comment:
So I just looked at how this function is actually being used...
In 131 cases it's simply `Scope.exit();`. In 1 case it's `Scope.exit(keep.toArray(new Key[keep.size()]));`. That's it. Clearly, the signature of the function is poorly designed, and it should be
`public static void exit(Iterable<Key> keep)` (i.e. don't return anything, and take a List instead of a plain array).